### PR TITLE
Changes required to compile Racket with C99+exts

### PR DESCRIPTION
**Pull request by pmatos**
With these changes RacketCS and RacketBC can be compiled with:
     `-std=c99 -D_DEFAULT_SOURCE`

### CODE REVIEW

**Review by mflatt — APPROVED** [review]
LGTM

**Comment by pmatos** [comment]
@mflatt before merging, can you check the snippet I added to the change? Is it still ok?

**Comment by pmatos** [comment]
> @mflatt before merging, can you check the documentation snippet I added to the change? Is it still ok?

**Comment by pmatos** [comment]
failing on windows. :) glad for CI in pull request.

**Comment by pmatos** [comment]
> failing on windows. :) glad for CI in pull request.

pushed fix

**Comment by mflatt** [comment]
Still ok as far as I can tell - thanks!

**Comment by pmatos** [comment]
Perfect, will wait until CI finishes to merge.

**Comment by pmatos** [comment]
All green - going in!

**Pull request merged**
